notes/roman-urdu/78_Surah_Naba.md

```diff
@@ -0,0 +1,44 @@
+# Number 78: Surah Naba
+
+Classification: Makki  
+Rukus: 2  
+Ayahs: 40  
+Subject: Qayyamat  
+Related Surah: Surah Mursalat  
+
+Background: ‌Surah Mursalat k baad discussion shru krdi thi qufaar ne k yeh kiya hisab qitab hoga etc.
+
+‌Yeh log jaan lengy us bari khabar (qayyamat) k bare m anqareeb
+‌Allah ki nematein:
+- ‌Zameen bichona;
+- pahar mekhen aor joro k shaklein;
+- neend thakan mitany k liye;
+- raat ko dhaanpne wali cheez;
+- din rozee qamany k liye;
+- 7 mazboot aasman;
+- suraj ek chiraag; baadal jin paani nichora jata;
+- jis k zareeye se sabza aor anaaj nikalta aor baghaat;
+- ‌Conclusion: in sab nemato ke baad kuch zemedari hai.
+
+‌Fesle ka din muqrar h.
+- ‌Jis din soor phoonka jaye ga.
+- sab foj dar foj.
+- Aasman khol dya jaye ga, us me darwazy ban jayengy.
+- Pahar rait ki misl bana dye jayengy.
+-
+‌Jahannam muntazr h apne eendhan k liye
+- jo k sarkashi walo ka h tikhana.
+- jahan unko kuch ni mily ga peene k lye sawai kholta howa pani ya peet.
+- inko tawawqo e ni thi hisab qitab ki.
+- azaab m kami ni ayegi.
+-
+‌Mutaqiyo k liye qayamabi hogi us din
+- baagh. anghoor. beewyaan. jaam.
+- Rab k inaam unke amal k wajha se.
+  
+‌Rab h wo asman zameen aor uske darmyan jo kuch h.  
+
+‌Us din koi kuch ni bole ga sawae jisko ijazat ho.   
+‌Yeh din atal hai.  
+Jiska khabardar kr dya gya.  
+‌Us din kafr kahe gy kaash hum miti hoty.     
```

Comment:
Same comments as last notes i.e.

1. Better to dived the whole notes doc into multiple headings based on topics and
2. have ordered list of points under them